config/bigip/waagent.conf

```diff
@@ -97,3 +97,6 @@ AutoUpdate.Enabled=y
 
 # Add firewall rules to protect access to Azure host node services
 OS.EnableFirewall=y
+
+# Enforce control groups limits on the agent and extensions
+CGroups.EnforceLimits=y
```

Comment:
Does BigIP have CGroups?

I guess it doesn't really matter too much.  If you can do this, you'll log a warning and carry on as usual.

Reply:
Figured it was better to assume it does.